Multisig/Logic/Models/Collectible/Collectible.swift

```diff
@@ -9,13 +9,13 @@
 import Foundation
 
 struct Collectible: Decodable, Hashable {
-    let address: AddressString?
+    let address: Address?
```

Comment:
why using Address here? I think, AddressString is the way  to go because it handles the JSON decoding properly